src/test/kotlin/racing/CarFactoryKoTest.kt

```diff
@@ -0,0 +1,21 @@
+package racing
+
+import io.kotest.core.spec.style.FreeSpec
+import io.kotest.matchers.shouldBe
+import racing.domain.CarFactory
+
+class CarFactoryKoTest : FreeSpec({
+    "createCars" - {
+        "with valid car names" - {
+            "return car list with input name" {
+                val names = listOf("Andy", "Bruce")
+
+                val racingCars = CarFactory().createCars(names)
+
+                names.forEachIndexed { index, name ->
+                    racingCars.cars[index].name shouldBe name
+                }
```

Comment:
```suggestion
                val actual = CarFactory().createCars(names).cars
                actual shouldContainExactlyInAnyOrder listOf(Car("Andy"), Car("Bruce"))
```
이건 어떠신지요. (테스트에서 로직을 최소화 하자는 피드백의 반복이네요 😓)